src/components/common/ButtonComponent.jsx

```diff
@@ -3,17 +3,29 @@ import { Button } from 'semantic-ui-react';
 import PropTypes from 'prop-types';
 
 const ButtonComponent = ({
-  buttonName, color, handleClick, buttonState, customCss
+  buttonName, color, handleClick, buttonState, customCss, disabledState
 }) => {
   if (color === 'primary') {
     return (
-      <Button primary onClick={handleClick} loading={buttonState} className={customCss}>
+      <Button
```

Comment:
I recommend a minor change to this component. The major differences between the props for primary and secondary color are:
- primary
- type
- loading

Therefore, we can have only one return. Something like:
```
const PRIMARY_COLOR = 'primary';

const ButtonComponent = ({
  buttonName,
  color,
  handleClick,
  buttonState,
  customCss,
  disabledState
}) => {
  let additionalProps = {
    secondary: true,
    type: "reset"
  }
  if (color === PRIMARY_COLOR) {
    additionalProps = {
      primary: true,
      loading: buttonState
    };
  }

  return (
    <Button
      {...additionalProps}
      onClick={handleClick}
      disabled={disabledState}
      className={customCss}
    >
      {buttonName}
    </Button>
  );
}
```

Reply:
Makes sense. 

Done.